### ISSUE TITLE
Submit a Sessionize CFP

### ISSUE THREAD

**Issue by Jandev** [jfversluis/CfpExchange#108]
While everybody probably loves submitting forms and filling out fields, it would be awesome if some CFP's could be filled out automatically.

A lot of CFP's are hosted on Sessionize, so when entering a CFP which is located at the hostname sessionize.com, choose the site-specific scraper and get the:
* CFP url
* Website URL
* Description
* Accommodation & Travel expenses
* ...

For a better UX, the field for the CFP URL also has to be moved to the top of the form.

**Comment by jfversluis** [comment]
Yeah, that's something I have been struggling with earlier. I then made the choice to go for the event URL as leading for some reason.

I guess it would make sense to take the CFP URL as a leading entity and get everything from there. Most of them would indeed probably come from Sessionize and/or maybe Papercall so it would be cool to scrape most from there, that will make filling in the form a lot easier probably.

**Comment by rickvdbosch** [comment]
Contacted Sessionize, unfortunately they currently don't have an API to get CfP information.

>Unfortunately, we don't have API for that at the moment. We do plan it for the future, but no specific dates yet. Sorry.
>
>Parsing seems like a good option for the moment, let us know if you need anything.

Tweet conversation [here](https://twitter.com/rickvdbosch/status/1273705976953257985).
